drivers/axi_core/jesd204/axi_adxcvr.c

```diff
@@ -501,16 +501,16 @@ int32_t adxcvr_init(struct adxcvr **ad_xcvr,
 
 	if (xcvr->lane_rate_khz && xcvr->ref_rate_khz) {
 		ret = adxcvr_clk_set_rate(xcvr, xcvr->lane_rate_khz, xcvr->ref_rate_khz);
-		if (ret)
-			goto err;//ToDo If it fails here, then you won't get its name in the caller, because following instr. not executed.
+		if (ret) {
```

Comment:
why adding brackets?

Reply:
Thank you for the observation. The brackets remained there after the development phase. Removed.